### PR TITLE
Thank and congrat people on getting a PR merged

### PR DESCRIPTION
**Pull request by KrauseFx**
screenshot 2017-03-27 09 04 45

The text might not be perfect yet, let me know how I can improve it. Also how do you like the label name?

### CODE REVIEW

File: bot.rb [hunk 1]
```diff
@@ -116,6 +117,30 @@ def process_closed_pr(pr, prs_to_releases)
...
+          congrats_on_merging << "Hey @#{pr.user.login} :wave:\n"
+          congrats_on_merging << "Thank you for your contribution to _fastlane_ and congrats on getting this pull request merged :tada:"
+          congrats_on_merging << "The code change now lives in the `master` branch, however it wasn't released to [RubyGems](https://rubygems.org/gems/fastlane) yet."
+          congrats_on_merging << "We usually ship about once a week, and your PR will be included in the next one.\n"
+          congrats_on_merging << "Please let us know if this change requires an immediate release by adding a comment here :+1:"
+          congrats_on_merging << "We'll notify you once we shipped a new release with your changes :rocket:"
+          client.add_comment(SLUG, pr.number, congrats_on_merging.join("\n"))
+          client.add_labels_to_an_issue(SLUG, pr.number, [INCLUDED_IN_NEXT_RELEASE])
+        end
+      end
```
Comment (mfurtak) [inline bot.rb]: This is great :+1: What do you think about extracting it into a method like the surrounding code?

**Comment by ohayon** [comment]
This is great @KrauseFx! I had a PR that I reverted that got this message, which is totally okay I think, but then also on the PR that did the reverting (https://github.com/fastlane/fastlane/pull/8667#issuecomment-289656967) I also got the message. I think that GitHub prefixes revert-PRs with `revert-`, can we add some logic in here to not comment if they PR branch begins with `revert-`?

File: bot.rb (hunk 1 above)
Comment (KrauseFx) [inline bot.rb]: Cool, will do

**Comment by KrauseFx** [comment]
@ohayon Seems like it's not worth extra complexity and work, only 2.5% of all PRs include the word `revert`, and even then it's correct that's it's not shipped at this point yet. (https://github.com/fastlane/fastlane/pulls?utf8=%E2%9C%93&q=is%3Apr%20revert%20 vs https://github.com/fastlane/fastlane/pulls)

**Pull request merged**